net/siproxd/src/opnsense/mvc/app/views/OPNsense/Siproxd/general.volt

```diff
@@ -86,6 +90,10 @@ POSSIBILITY OF SUCH DAMAGE.
                 </tr>
             </tfoot>
         </table>
+        <hr />
```

Comment:
&lt;hr /&gt; belongs into the div below, alignment is usually fixed with &lt;br /&gt;&lt;br /&gt; after the button in other forms